private-templates-service/client/src/components/NavBar/NavBar.tsx

```diff
@@ -96,14 +115,15 @@ const NavBar = (props: NavBarProps) => {
     default:
       return (
         <Banner>
-          <MobileBanner>
+          <BaselineBanner>
```

Comment:
why do we need `BaselineBanner`? Seems like the only difference is `align-items: baseline;`

Reply:
the pencil icon is a few pixels off without that style lol